### ISSUE TITLE
Accessibility issues to address

### ISSUE THREAD

**Issue by hatfieldjm4** [GSA/GEAR3#1076]
- [ ] On report views, add accessible name to the magnifying glass icon in the table search 

<img width="166" height="236" alt="Image" src="https://github.com/user-attachments/assets/54bc46f4-7014-455c-afb5-50ba43bda386" />

- [ ] Add alt-text to the footer image (GSA flag) on the usa-banner

<img width="163" height="167" alt="Image" src="https://github.com/user-attachments/assets/b8d19747-8854-48e2-8ad6-355d99486959" />

- [ ] Make links in the footer accessible (info [here](https://dequeuniversity.com/rules/axe/4.11/link-name))

<img width="170" height="149" alt="Image" src="https://github.com/user-attachments/assets/136daef8-7b10-40ba-98db-a66433833809" />

- [ ] Change contrast of text/background colors for "status" fields (e.g. the IT standards "status" and "approved versions" field both in the report view and the detail view, the business systems "status" field in the detail view, the FISMA system "status" field in the detail view, the "status" field on the dashboard's "10 most recent GSA IT standards" table)
 
<img width="167" height="263" alt="Image" src="https://github.com/user-attachments/assets/46654004-bf94-486c-96c4-4183ec66c21f" />

- [ ] On the sidebar, some elements have a [tabindex] value greater than 0 

<img width="162" height="371" alt="Image" src="https://github.com/user-attachments/assets/f82f7312-e007-4d03-9f41-45f39684ae78" />

- [ ] Document does not have a main [landmark](https://dequeuniversity.com/rules/axe/4.11/landmark-one-main).

<img width="169" height="185" alt="Image" src="https://github.com/user-attachments/assets/01d6a343-d042-4c61-9f52-df41f0acb345" />

**Comment by shahasshrestha** [comment]
Solution Merged to FwdDev:
 https://github.com/GSA/GEAR3/pull/1073
https://github.com/GSA/GEAR3/pull/1079
